### PR TITLE
Add help information for gpfdist

### PR DESCRIPTION
**Pull request by Hlinbit**
Add some help information for the gpfdist flag 'ssl_verify_peer'.

### CODE REVIEW

File: gpMgmt/doc/gpfdist_help [hunk 1]
```diff
@@ -169,6 +170,17 @@ OPTIONS
  to use this flag in conjunction with '--compress'. However, when used 
  together, the effect is equivalent to using '--multi_thread' by itself.
 
+--ssl_verify_peer <on/off>
+
+  This option is used to configure the SSL authentication files required in 
+  the GPFDIST protocol, as well as to control the behavior of GPFDIST regarding 
+  SSL identity authentication.
```
Comment (hyongtao-db) [inline gpMgmt/doc/gpfdist_help:177]: Case senstitive? I didn't find any capital case `GPFDIST` in this markdown.

**Review by hyongtao-db — APPROVED** [review]
Looks good. Just a simple question.

**Review by RuiWang-vmw — APPROVED** [review]

File: gpMgmt/doc/gpfdist_help [hunk 2]
```diff
@@ -169,6 +170,17 @@ OPTIONS
...
+
+  This option is used to configure the SSL authentication files required in 
+  the gpfdists protocol, as well as to control the behavior of gpfdist regarding 
+  SSL identity authentication.
+  
+  If the flag is 'on', gpfdist will be forced to check the identity of clients, 
+  and CA certification file (root.crt) is necessary to be laid in the SSL 
+  certification directory. If the flag is 'off', gpfdist will not check the 
+  identity of clients, and it doesn't matter whether the CA certification file 
+  exists on the gpfdist side.
```
Comment (zhaorui20005) [inline gpMgmt/doc/gpfdist_help:183]: Do we need to add to state the default value of this parameter?

Comment (Hlinbit) [reply gpMgmt/doc/gpfdist_help:183]: I have added the information.

Comment (zhaorui20005) [reply gpMgmt/doc/gpfdist_help:183]: So the default value is on or off?

Comment (zhaorui20005) [reply gpMgmt/doc/gpfdist_help:183]: Ok, I see it. It is better to append commit instead of force push.

**Review by zhaorui20005 — APPROVED** [review]

**Pull request merged**